protocol_details/rpmsg_comms.rst

```diff
@@ -2,24 +2,28 @@
 RPMsg Communication Flow
 =========================
 
-The following figure describes the sequence used for transaction of a RPMsg message from one core to
-the other.
-The sequence differs according to the roles of core A and core B. In the figure above, core A is the
-Master and core B is the Remote. The Master core allocates buffers used for the transmission from
-the “used” ring buffer of a vring, writes RPMsg Header and application payload to it and then
-enqueues it to the “avail” ring buffer.
+The following figure describes the sequence used for transaction of a RPMsg message from one core
+to the other.
+The sequence differs according to the roles of core A and core B. In the figure above, core A is
+the main controller and core B is the remote. The main controller allocates buffers used for
+the transmission from the “used” ring buffer of a vring, writes RPMsg Header and application
+payload to it and then enqueues it to the “avail” ring buffer.
```

Comment:
proposal to apply on each step to clarify the VRING 0/1
To transmits a RPmsg message, the main controller:
- gets a transmission buffer from “used” queue of the VRING0
- fills RPMsg Header and payload
- enqueues the RPmsg to the “avail” queue of the VRING0.